### ISSUE TITLE
[Elements/inputs] Reconsider the use of placeholder text in input boxes

### ISSUE THREAD

**Issue by AccessForAll** [18F/govt-wide-patternlibrary#168]
* Check the placeholder text contrast (the contrast is currently 2.88:1, but that may not be the right color).

* Also, placeholder text performs unpredictably for some assistive technology users. If the text fails to disappear after people insert their cursors into the input box, or it receives focus, people who are visually impaired may not know that the placeholder text will be submitted along with what they type into the box. 

* Reconsider the use of placeholder text. Proper text labels above fields should provide sufficient instructional content and context.

* Reference:
     * Placeholder Text Is Harmful, Nielsen Norman Group: http://nngroup.com/articles/form-design-placeholders/
     * Placeholder Attribute Is Not a Label, Web Axe: http://www.webaxe.org/placeholder-attribute-is-not-a-label/

**Comment by maya** [comment]
Related: https://github.com/18F/govt-wide-patternlibrary/issues/172#issuecomment-119207401

We have to discuss at large at whether or not we should be recommending placeholder text at all. See alternate idea here: https://github.com/18F/govt-wide-patternlibrary/issues/117#issuecomment-119069873

**Comment by colinpmacarthur** [comment]
We've addressed this issue for footers in #172 and I think #117 came to a useful conclusion about alternatives to placeholder text in general. Thus, I'm closing this issue. 

@AccessForAll: if you see other patterns with placeholder text, feel free to submit another issue specifically for each one. Many thanks!
